### DISCUSSION TITLE
PDF preview text selection

### DISCUSSION THREAD

**Discussion by lcristeli** [QL-Win/QuickLook#1384] [Ideas]
When previewing some PDF files (normal ones and scans with OCR) I need to select a text to copy, but text selection is not available. Can this feature be enabled or included?

Anyway, I would like to leave my appreciation for the awesome program you guys made.

Thanks

**Comment by emako** [comment]
It is recommended to use QuickLook.Plugin.PdfViewer-Native as an alternative

https://github.com/QL-Win/QuickLook.Plugin.PdfViewer-Native
